packages/vitest/src/node/pool.ts

```diff
@@ -36,6 +37,13 @@ export function createPool(ctx: Vitest): WorkerPool {
 
     maxThreads,
     minThreads,
+
+    execArgv: [
+      '--experimental-loader',
+      loaderPath,
+      ...ctx.server.config.resolve.conditions?.flatMap(c => ['-C', c]) ?? [],
+      '--no-warnings',
```

Comment:
Is no warnings bypass all warnings from Node? Or can it be selective?

Reply:
I don't think you can select what warning to suppress. Here I am suppressing `experimental` warning.

Reply:
We can do something like this:

```ts
process.removeAllListeners('warning')
process.addEventListener('warning', e => !e.message.includes('experimental') && process.stdout.write(e.message))
```

Reply:
Ref: how `tsx` handles it
https://github.com/esbuild-kit/tsx/blob/develop/src/suppress-warnings.cts